mypy/modulefinder.py

```diff
@@ -472,15 +466,53 @@ def make_abspath(path: str, root: str) -> str:
         site_packages = ast.literal_eval(
             subprocess.check_output([python_executable, sitepkgs.__file__],
             stderr=subprocess.PIPE).decode())
-    egg_dirs = []
+    return expand_site_packages(site_packages)
+
+
+def expand_site_packages(site_packages: List[str]) -> Tuple[List[str], List[str]]:
+    """Expands .pth imports in site-packages directories"""
+    egg_dirs = []  # type: List[str]
     for dir in site_packages:
-        pth = os.path.join(dir, 'easy-install.pth')
-        if os.path.isfile(pth):
-            with open(pth) as f:
-                egg_dirs.extend([make_abspath(d.rstrip(), dir) for d in f.readlines()])
+        if not os.path.exists(dir):
```

Comment:
I'd use `isdir` here, on the odd chance that somehow it exists as a file, since the `listdir` operation below would raise in that case.

Reply:
Good catch. I'll update that!